pygmt/accessors.py

```diff
@@ -9,20 +9,43 @@
 @xr.register_dataarray_accessor("gmt")
 class GMTDataArrayAccessor:
     """
-    This is the GMT extension for :class:`xarray.DataArray`.
+    GMT extension for :class:`xarray.DataArray`.
 
-    You can access various GMT specific metadata about your grid as follows:
+    The extension provides easy ways to access and change the GMT specific
+    properties about grids. Currently, two properties are available:
+
+    - ``registration``: Gridline (0) or Pixel (1) registration
+    - ``gtype``: Cartesian (0) or Geographic (1) coordinate system
+
+    You can access these GMT specific properties about your grid as follows:
 
     >>> from pygmt.datasets import load_earth_relief
     >>> # Use the global Earth relief grid with 1 degree spacing
     >>> grid = load_earth_relief(resolution="01d", registration="pixel")
-
     >>> # See if grid uses Gridline (0) or Pixel (1) registration
     >>> grid.gmt.registration
     1
     >>> # See if grid uses Cartesian (0) or Geographic (1) coordinate system
     >>> grid.gmt.gtype
     1
+
+    You can also set the GMT specific properties for grids created by yourself:
+
+    >>> import numpy as np
```

Comment:
Note: this inline example is modified from https://github.com/GenericMappingTools/pygmt/issues/390.

Reply:
ok